packages/devtools_app/lib/src/timeline/html_frames_bar_chart.dart

```diff
@@ -207,7 +207,7 @@ class PlotlyDivGraph extends CoreElement {
       // TODO(terry): HACK - Ignore the event.
       log(
         'Ignored onFrameAdded - bad data.\n [uiDuration: '
-        '${frame.uiDuration}, gpuDuration: ${frame.gpuDuration}',
+        '${frame.uiDuration}, gpuDuration: ${frame.rasterDuration}',
```

Comment:
`gpuDuration` ==> `rasterDuration`?

Reply:
whoops - this change wasn't supposed to be there. Removed. We didn't have a type on `frame` so the analyzer didn't check it.